pkgs/applications/virtualization/gvisor/default.nix

```diff
@@ -120,5 +120,7 @@ in buildBazelPackage rec {
     license = licenses.asl20;
     maintainers = with maintainers; [ andrew-d ];
     platforms = [ "x86_64-linux" ];
+    # the version we have right now does not compile with go 1.17
+    broken = true;
```

Comment:
latest builds with go1.17 - lets bump before landing this PR. https://github.com/google/gvisor/blob/master/go.mod#L3

Reply:
Can you bump it? I think I tried and it was non-trivial. IMO this is the maintainer's (@andrew-d) job not ours.

The release is ~10 months out of date, with a software that releases _very often_, looking at https://github.com/google/gvisor/tags.